Hexadecimal (#1762)
Codecademy/docs: content/general/concepts/hexadecimal/hexadecimal.md

---
Title: 'Hexadecimal'
Description: 'Hexadecimal(hex, hex numbers, or base-16) is a numeral system that uses a base of 16 as opposed to the standard decimal system, base 10.'
Subjects:
  - 'Web Development'
  - 'Web Design'
  - 'Computer Engeneering'
Tags:
  - 'colors'
CatalogContent:
  - 'paths/computer-science'
  - 'paths/code-foundations'
---

**Hexadecimal** (hex, hex numbers, or base-16) is a number system that uses a base of 16 as opposed to the standard decimal system, which is base 10. When representing a number, both number systems share the first ten digits, 0-9. However, for hexadecimal, the remaining digits 10-15 are represented by the letters A-F (not case sensitive)

| Decimal | Hex |
| ------- | --- |
| 1       | 1   |
| 2       | 2   |
| 3       | 3   |
| 4       | 4   |
| 5       | 5   |
| 6       | 6   |
| 7       | 7   |
| 8       | 8   |
| 9       | 9   |
| 10      | A   |
| 11      | B   |
| 12      | C   |
| 13      | D   |
| 14      | E   |
| 15      | F   |

## Power Of 2

Hexadecimal, as well as binary (base 2) and octal (base 8), have bases that are powers of two, making it easy to convert between one another.

## Syntax

> A hexadecimal number is prefixed with `0x` when refering to the base 16 number system.

Reviewer: ```suggestion
A hexadecimal number is prefixed with `0x` in many computer languages to identify it as a base 16 number.
```

Reviewer: I updated it for you,

---
Title: 'Hexadecimal'
Description: 'Hexadecimal(hex, hex numbers, or base-16) is a numeral system that uses a base of 16 as opposed to the standard decimal system, base 10.'
Subjects: 
    - 'Web Development'
    - 'Web Design'
    - 'Computer Engeneering'
Tags:
    - 'colors'
CatalogContent:
    - 'paths/computer-science'
    - 'paths/code-foundations'
---

> **Hexadecimal** (hex, hex numbers, or base-16) is a numeral system that uses a base of 16 as opposed to the standard decimal system, base 10. The two have the first ten digits in common, 0-9. The remaining digits 10-15 are represented by the letters A-F in base 16. The letters are case-insensitive. 

Reviewer 2: ```suggestion
**Hexadecimal** (hex, hex numbers, or base-16) is a number system that uses a base of 16 as opposed to the standard decimal system, which is base 10. When representing a number, both number systems share the first ten digits, 0-9. However, for hexadecimal, the remaining digits 10-15 are represented by the letters A-F (not case sensitive)
```

| Decimal   | Hex       |
|---------- |-----------|
|     1     |     1     |    
|     2     |     2     |
|     3     |     3     |
|     4     |     4     |
|     5     |     5     |
|     6     |     6     |
|     7     |     7     |
|     8     |     8     |
|     9     |     9     |
|    10     |     A     |
|    11     |     B     |
|    12     |     C     |
|    13     |     D     |
|    14     |     E     |
|    15     |     F     |

## Power Of 2

> Hexadecimal, as well as binary(base 2) and octal(base 8) number systems, have bases that are powers of 2. This makes it easy to convert between these systems. 

Reviewer 2: ```suggestion
Hexadecimal, as well as binary (base 2) and octal (base 8), have bases that are powers of two, making it easy to convert between one another.
```

## Syntax

A hexadecimal number is prefixed with `0x`.

## Hex Colors

> Hex numbers often represent colors(hex colors) and are specified with `#RRGGBB`(red, green, and blue). One example of a hex color is `#0000FF` and represents the color blue. We know this because red and green is set to their lowest value, `00`, and blue set to their highest value, `FF`. A hex color is prefixed with `#`.

Reviewer 2: ```suggestion
Hex numbers are often used to represent colors (hex colors) and are specified in the following way: `#RRGGBB`(red, green, and blue). One example of a hex color is `#0000FF` which represents the color blue. This is because red and green are set to their lowest value, `00`, and blue is set to its highest value, `FF`.
```

---
Title: 'Hexadecimal'

> Description: 

Reviewer: We're missing a description in the metadata. (You can paraphrase the first sentence of your entry.)

Subjects: 
    - 'Web Development'
    - 'Web Design'
    - 'Computer Engeneering'
Tags:

>     - 'color'

Reviewer: Tags need to come form the list in the [`tags.md` file in the `/documentation` folder.](https://github.com/Codecademy/docs/blob/main/documentation/tags.md)

    - 'bases'
CatalogContent:

>     - 'paths/computer-science-career-path'

Reviewer: ```suggestion
    - 'paths/computer-science'
    - 'paths/code-foundations'
```

---

> 

Reviewer: ```suggestion
```
Remove extra line break.

> `Hexadecimal`(hex, hex numbers, or base-16) is a numeral system that uses a base of 16 as opposed to the standard decimal system, base 10. The two have the first ten digits in common, 0-9. The remaining digits 10-15 are represented by the letters A-F in base 16. The letters are case-insensitive. 

Reviewer: ```suggestion
**Hexadecimal** (hex, hex numbers, or base-16) is a numeral system that uses a base of 16 as opposed to the standard decimal system, base 10. The two have the first ten digits in common, 0-9. The remaining digits 10-15 are represented by the letters A-F in base 16. The letters are case-insensitive. 
```

| Decimal   | Hex       |
|---------- |-----------|
|     1     |     1     |    
|     2     |     2     |
|     3     |     3     |
|     4     |     4     |
|     5     |     5     |
|     6     |     6     |
|     7     |     7     |
|     8     |     8     |
|     9     |     9     |
|    10     |     A     |
|    11     |     B     |
|    12     |     C     |
|    13     |     D     |
|    14     |     E     |
|    15     |     F     |

> ## Syntax

Reviewer: Remove this syntax section as it's not generally applicable.

A hexadecimal number is prefixed with `0x`.

> ## Hex Colors

Reviewer: ```suggestion
## Hex Colors

```
line break after header.
